browser/ui/views/tabs/brave_tab_group_header.cc

```diff
@@ -40,6 +43,25 @@ SkColor GetGroupBackgroundColorForVerticalTabs(
 
 BraveTabGroupHeader::~BraveTabGroupHeader() = default;
 
+void BraveTabGroupHeader::AddedToWidget() {
+  TabGroupHeader::AddedToWidget();
+  if (!ShouldShowVerticalTabs()) {
+    return;
+  }
+
+  if (editor_bubble_tracker_.is_open()) {
+    auto* bubble_delegate = editor_bubble_tracker_.widget()
+                                ->widget_delegate()
+                                ->AsBubbleDialogDelegate();
+    DCHECK(bubble_delegate);
+    // Technically, this can happen when tab strip's orientation changes
+    // with the editor bubble open. It re-parents the widget which can cause
+    // DCHECK failure. We should call SetAnchorView again to reset
+    // the anchor widget.
+    bubble_delegate->SetAnchorView(this);
+  }
+}
```

Comment:
@simonhong Do you mind take another look? This part was added to fix test failure on Linux.